### PR TITLE
Move Slider component styles into styles.js

### PR DESCRIPTION
**Pull request by dinartecc**
Also fixed Accordion not properly displaying ArrowDown icon

### CODE REVIEW

File: src/components/Slider/index.js [hunk 1]
```diff
@@ -106,17 +78,19 @@ const Slider = ({
...
-      draggable={draggable}
-      nextArrow={<NextArrow />}
-      prevArrow={<PrevArrow />}
-      speed={speed}
-      {...customPagingProp}
-      {...props}
-    >
-      {children}
-    </SlickSlider>
+    <Box sx={slickStyles}>
```
Comment (luisduenas) [inline src/components/Slider/index.js]: Can we change this to ```<Box __css={slickStyles}>``` ?

**Review by luisduenas — COMMENTED** [review]
LGTM

**Comment by hypervillain** [comment]
Nice! The css import broke with Nextjs

**Comment by jmolivas** [comment]
Can we have the list of steps you are following to test it so we can try it locally

**Comment by hypervillain** [comment]
I made a sample app here: https://github.com/prismicio/sm-next-app-example. It uses the Next SliceZone and an `essential-slices` package (current components minus the CSS slick file import)

**Pull request merged**